decoder/src/lib.rs

```diff
@@ -96,11 +105,21 @@ impl StringEntry {
     }
 }
 
+/// Data that uniquely identifies a `defmt::bitflags!` invocation.
+#[derive(Debug, PartialEq, Eq, Hash)]
+struct BitflagsKey {
+    /// Name of the bitflags struct (this is really redundant with `disambig`).
```

Comment:
does it need to be in the key? (as in can we remove it? or are using the `ident` in the key in the formatting?). package + disambiguator should be enough to uniquely identify this in the entire dependency graph, right?

Reply:
Yeah, I mostly added that because I don't really like that we rely on `Span`'s hash identity in the disambiguator field, and wanted to add something more than that